### PR TITLE
fix incorrect version comparision

### PR DESCRIPTION
**Pull request by Zefau**
In addition to https://github.com/ioBroker/ioBroker.admin/pull/464/ a fix for a version comparison with respect to semver.

### CODE REVIEW

**Comment by Zefau** [comment]
@GermanBluefox @Apollon77 Would you guys merge this ? I have tested this locally and it works fine for me. This is in alignment with the pr https://github.com/ioBroker/ioBroker.admin/pull/464. I think the function I have fixed there is not used?

**Comment by GermanBluefox** [comment]
Thank you!

File: src/js/admin.js [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /* global _ */
 /* global ace */
 /* global console */
+/* global semver */
```
Comment (AlCalzone) [inline src/js/admin.js:11]: @Zefau Just out of curiosity: This is the frontend, isn't it? How is `semver` loaded there?

Comment (Zefau) [reply src/js/admin.js:11]: Don't know how that works, but I tested it successfully. Maybe you could adapt the change and verify?

Comment (Apollon77) [reply src/js/admin.js:11]: Ok, can this now be released as a version or not?

**Pull request merged**